docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,282 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
+## Athena
+
+If you are running Teleport on AWS, you can use an
+[Athena](https://aws.amazon.com/athena/)-based audit log system that manages
+[Parquet files](https://parquet.apache.org/) stored on
+[S3](https://aws.amazon.com/s3/) as a storage back-end to achieve high
+availability. Athena backend supports only one type of Teleport data, audit log
+events.
+
+The Athena audit logs back-end provides better scalability and searchability in
+comparison to DynamoDB.
+
+Athena audit logs are eventually consistent. This means that events won't be
+immediately displayed in the Teleport Web UI. It can take up to 1 minute until
+they are visible (depending on the `batchMaxInterval` setting and event load).
+
+### Infrastructure setup
+
+The Auth service uses an SQS queue subscribed to an SNS topic for event
+publishing. A single Auth Service instance reads events in batches from SQS,
+converts them into Parquet format, and sends the resulting data to S3. During
+queries, the Athena engine searches for events on S3, reading metadata from a
+Glue table.
+
+You can set up the required infrastructure to support the Athena backend with
+the following Terraform script:
+
+<Details title="Terraform script" opened={false}>
+```
+Add here terraform examples when they are merged.
+```
+</Details>
+
+### Configuring the Athena audit log backend
+
+To configure Teleport to use Athena:
+
+- Make sure you are using **Teleport version 14.0.0** or newer.
+- Prepare infrastructure
+- Specify an Athena URL inside the `audit_events_uri` array in your Teleport
+  configuration file:
+
+```yaml
+teleport:
+  storage:
+    # This setting configures Teleport to keep a copy of the audit log in Athena
+    # and a copy on a local filesystem, and also to output the events to stdout.
+    audit_events_uri:
+      # More details about the full Athena url are shown below.
+      - 'athena://database.table?params'
+      - 'file:///var/lib/teleport/audit/events'
+      - 'stdout://'
+```
+
+```
+athena://db.table?topicArn=arn:aws:sns:region:account_id:topic_name&largeEventsS3=s3://transient/large_payloads&locationS3=s3://long-term/events&workgroup=workgroup&queueURL=https://sqs.region.amazonaws.com/account_id/queue_name&queryResultsS3=s3://transient/query_results
+```
+
+The URL hostname consist of `database.table`, which points to the Glue database
+and a table which will be used by the Athena audit logger.
+
+Other parameters are specified as query parameters to Athena url.
+
+The following parameters are required:
+- `topicArn=arn:aws:sns:region:account_id:topic_name` - ARN of SNS topic where events are published
```

Comment:
Are the values after `=` defaults or examples? I think this would be clearer as a table with three columns:
- Parameter name
- Example (or default, if the values after `=` are defaults)
- Description

Reply:
I have rework it to use table, also optional parameters